app/src/main/java/com/cmput301f20t21/bookfriends/ui/request/RequestAdapter.java

```diff
@@ -0,0 +1,86 @@
+package com.cmput301f20t21.bookfriends.ui.request;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.cmput301f20t21.bookfriends.R;
+import com.cmput301f20t21.bookfriends.entities.Request;
+
+import java.util.ArrayList;
+
+public class RequestAdapter extends RecyclerView.Adapter<RequestAdapter.RequestHolder> {
+    private ArrayList<Request> requestItems;
+    private OnItemClickListener listener;
+    public interface OnItemClickListener{
+        void onRejectClick(int position);
+        void onAcceptClick(int position);
+    }
+
+    public void setOnItemClickLisener(OnItemClickListener listener) {
+        this.listener = listener;
+    }
+
+    public static class RequestHolder extends RecyclerView.ViewHolder {
```

Comment:
I think the convention we're following is to name this `ViewHolder` as well?
So it will be `public static class ViewHolder extends RecyclerView.ViewHolder`